puppet/modules/freight/manifests/user.pp

```diff
@@ -57,8 +58,12 @@
     },
   ]
 
-  # locations doesn't autorequire the headers module
-  include apache::mod::headers
+  # vhosts don't autorequire the expires module
+  # https://github.com/puppetlabs/puppetlabs-apache/pull/2559
+  # limit to not EL7 as there we use apache::default_mods
+  if $facts['os']['family'] != 'RedHat' or $facts['os']['release']['major'] != '7' {
+    include apache::mod::expires
+  }
```

Comment:
Or we move that to
https://github.com/theforeman/foreman-infra/blob/78d1281d0ee9c1326d6939d1cea993ef74e3e970/puppet/data/osfamily/RedHat-9.yaml#L2-L3